schemas/maas-backend/customers/verification/initiate/response.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/customers/verification/initiate/response.json",
+  "description": "MaaS customer verification initiate",
+  "type": "object",
+  "properties": {
+    "customer": {
+      "type": "object",
+      "properties": {
+        "verification": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
```

Comment:
Can some of these property definitions be moved to file level definitions? It's quite hard to read so deeply nested definitions :p